CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v.13.1] - 2019-07-22
```

Comment:
At the bottom of this doc, add a link for the comparison between the versions.

Reply:
👍  I hadn't run through the deploy doc yet. 

150235450